### PR TITLE
Apply modification for the meta description.

### PR DESCRIPTION
**Pull request by andizer**
Also apply the modification for the meta description.

See: https://github.com/Yoast/wordpress-seo/pull/3741

### CODE REVIEW

File: js/snippetPreview.js [hunk 1]
```diff
@@ -428,6 +428,11 @@ SnippetPreview.prototype.formatMeta = function() {
 	// Cut-off the meta description according to the maximum length
 	meta = meta.substring( 0, YoastSEO.analyzerConfig.maxMeta );
 
+	// Apply modification to the desc before showing it.
```
Comment (atimmer) [inline js/snippetPreview.js]: Shouldn't we apply the modifications before we strip all tags and cut it off to the maximum length? The modification might add HTML or content we still want to strip and cut off.

**Comment by atimmer** [comment]
CR Done 👍🏻

**Comment by atimmer** [comment]
CR2 Done 👍🏻

**Comment by jcomack** [comment]
Acceptance done

**Pull request merged**